xml/obs_best_practice_webui_usage.xml

```diff
@@ -155,10 +155,24 @@
    Now let's create a link to a package and add a repository to build
    against. A link is basically a pointer to sources of an already existing
    package in another project. By *repository*, we mean a build target -
-   like Trunk/Fedora_12/openSUSE_11.4. Let's follow these steps: 1. Add link
-   to the existing package. 2. Add repository / build target. 3. Enable the
-   build (optional, if not already enabled). 4. Observe the build on the
-   monitor page 5. Look at package's page
+   like Trunk/Fedora_12/openSUSE_11.4. Let's follow these steps:
```

Comment:
By "repository" we mean container of built binary packages like Debian_8 or openSUSE_13.2 .